src/local-settings.js

```diff
@@ -50,6 +51,20 @@ export function setIpfsGateway(gateway) {
   return setLocalSetting(IPFS_GATEWAY, gateway)
 }
 
+// The previous package version is stored in localStorage,
```

Comment:
Are we already injecting the package version into the build environment elsewhere?

Reply:
Added the package version in all build configs now